### PR TITLE
Babel config

### PR DESCRIPTION
**Pull request by marcelomorgado**
refs #22 and #23 - not tried out w/ react-native yet.

### CODE REVIEW

**Comment by marcelomorgado** [comment]
@babel/runtime should be installed to can be used by `tasit` react-native demo

File: packages/tasit-sdk/package.json [hunk 1]
```diff
@@ -16,12 +17,21 @@
...
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
+    "prepublish": "npx babel src --out-dir lib --ignore **/*.test.js",
+    "test": "npx mocha src/index.test.js --require @babel/register"
   },
   "bugs": {
     "url": "https://github.com/tasitlabs/tasitsdk/issues"
   },
   "dependencies": {
-    "tasit-account": "^0.0.1"
```
Comment (pcowgill) [inline packages/tasit-sdk/package.json]: @marcelomorgado So you're thinking we'll reintroduce this dependency once we publish it, right?

File: packages/tasit-sdk/src/index.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Account } from "./index.js";
```
Comment (pcowgill) [inline packages/tasit-sdk/src/index.test.js]: @marcelomorgado I usually leave off the `.js` suffix. Just a style thing though - either way works.

File: packages/tasit-sdk/lib/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = exports.Account = void 0;
+
+var _wallet = require("ethers/wallet");
+
+const create = async () => {
```
Comment (pcowgill) [inline packages/tasit-sdk/lib/index.js]: @marcelomorgado Do you think it matters that we have this as an async function when `Wallet.createRandom` from ethers isn't?

File: packages/tasit-sdk/src/index.js [hunk 4]
```diff
@@ -1,14 +1,22 @@
 "use strict";
 
-// TODO: Add Babel support
-
 import { Wallet } from "ethers/wallet";
 
-export const create = async () => {
+const create = async () => {
```
Comment (pcowgill) [inline packages/tasit-sdk/src/index.js]: @marcelomorgado Do you think it matters that we have this as an async function when Wallet.createRandom from ethers isn't?

**Comment by pcowgill** [comment]
@marcelomorgado I'm going to merge this now, but I left a few comments I'd like you to check out for future PRs. Thanks!

File: packages/tasit-sdk/src/index.js (hunk 4 above)
Comment (marcelomorgado) [reply packages/tasit-sdk/src/index.js]: No, not really.

File: packages/tasit-sdk/package.json [hunk 5]
```diff
@@ -16,12 +17,21 @@
     "url": "git+https://github.com/tasitlabs/tasitsdk.git"
   },
   "scripts": {
-    "test": "echo \"Error: run tests from root\" && exit 1"
+    "prepublish": "npx babel src --out-dir lib --ignore **/*.test.js",
```
Comment (pcowgill) [inline packages/tasit-sdk/package.json]: @marcelomorgado  "As of npm@5, `prepublish` scripts are deprecated. Use `prepare` for build steps and `prepublishOnly` for upload-only."

**Pull request merged**